ntbs-service/Models/PatientTBHistory.cs

```diff
@@ -9,9 +9,9 @@ namespace ntbs_service.Models
     [Owned]
     public class PatientTBHistory : ModelBase
     {
-        public bool NotPreviouslyHadTB { get; set; }
+        public bool? NotPreviouslyHadTB { get; set; }
         
-        [RequiredIf("ShouldValidateFull && !NotPreviouslyHadTB", ErrorMessage = ValidationMessages.TBHistoryIsRequired)]
+        [RequiredIf("ShouldValidateFull && NotPreviouslyHadTB == false", ErrorMessage = ValidationMessages.TBHistoryIsRequired)]

```

Comment:
Is this change to avoid seeing `null` as "false"? is so, a comment might be worth it, o/w a helpful soul might see this and "fix" ;)